src/app/utils/socket-utils.ts

```diff
@@ -427,6 +429,7 @@ export function getSocketsByType(
 export function getWeaponSockets(
   item: DimItem,
   excludeEmptySockets = false,
+  includeFakeMasterwork = false,
```

Comment:
Is there a place we wouldn't want to include the fake masterwork?

Reply:
Organizer, I think.